templates/encrypt.nginx.tmpl

```diff
@@ -10,11 +10,27 @@ server {
     listen [::]:443 ssl http2 default_server;
     ssl_certificate {{fullchain}};
     ssl_certificate_key {{privkey}};
+    ssl_dhparam {{dhparam}};
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2;
+    ssl_prefer_server_ciphers on;
+    ssl_ciphers "EECDH+AESGCM:EDH+AESGCM:AES256+EECDH:AES256+EDH";
```

Comment:
It might be best to provide more comments about these config options.

- Are they best-practices? if so, state so and provide a link to the source so we know where to look for updates in a few months time.
- Do they solve any bugs/issues? If so, state that, so we know what to change/what to keep in a few months time.